wiki.py

```diff
@@ -512,6 +512,13 @@ def _get_programs(self):
             self._make_year_title(
                 self._config["year_pages"]["operational_plan"])
         )
+        if not operational_plan_page.exists():
+            title = operational_plan_page.title()
+            raise Exception(
+                f"Page '{title}' doesn't exist and is required to create "
+                "this page."
```

Comment:
When the exception is raised it is not clear what "this page" references.

You probably need to let the exception bubble up one level and be re-raised where this context is known.